src/index.js

```diff
@@ -14,10 +19,10 @@ const renderMenu = root => {
         <li>
           <a href="#${i + 1}">
             <span class="material-symbols-outlined">
-              ${Math.random() > 0.5 ? "check_box_outline_blank" : "check_box"}
+              ${(localStorage.getItem(i + 1) == null) ? "check_box_outline_blank" : "check_box"}
```

Comment:
Since `localStorage` isn't user-facing, I'd use 0-indexing.

Always use `===`, `==` is broken in JS.

Reply:
I would use keys like:

```
{
  "bestScorePerLevel": {1: "42", ...}
}
```

or 

```
{
  "level-1": "42",
  ...
}
```

Reply:
For the localStorage, let's keep it the way it is for now. The reason why I didn't use "level-1" as keys coz then I have to parse this and extract the actual level numbers. Right now I can directly access the key values without intermediary conversions.

Reply:
Can you implement what you are suggesting? I don't know how to do that.

Reply:
We can ignore this for now.